src/phases/tpch/denormalized/AvgAcctBal.yml

```diff
@@ -0,0 +1,44 @@
+SchemaVersion: 2018-07-01
```

Comment:
I could put all of these in an `artifical` subdirectory or somewhere else entirely if we want them separated from the standard queries. Wasn't sure where the best spot would be.